--- a/tests/test-umockdev-run.vala
+++ b/tests/test-umockdev-run.vala
@@ -231,7 +231,7 @@
 #if HAVE_SELINUX
     // we may run on a system without SELinux
     try {
-        Process.spawn_command_line_sync ("command -v selinuxenabled", null, null, out exit);
+        Process.spawn_command_line_sync ("selinuxenabled", null, null, out exit);
     } catch (SpawnError e) {
         exit = 1;
     }
@@ -374,7 +374,7 @@
 #if HAVE_SELINUX
     // we may run on a system without SELinux
     try {
-        Process.spawn_command_line_sync ("command -v selinuxenabled", null, null, out exit);
+        Process.spawn_command_line_sync ("selinuxenabled", null, null, out exit);
     } catch (SpawnError e) {
         exit = 1;
     }
--- a/tests/test-umockdev-vala.vala
+++ b/tests/test-umockdev-vala.vala
@@ -205,7 +205,7 @@
 {
   int exit;
   try {
-      Process.spawn_command_line_sync ("command -v selinuxenabled", null, null, out exit);
+      Process.spawn_command_line_sync ("selinuxenabled", null, null, out exit);
   } catch (SpawnError e) {
       exit = 1;
   }
